.github/workflows/node.js.yml

```diff
@@ -0,0 +1,87 @@
+name: Deploy Testing
+on:
+  push:
```

Comment:
Does this work for merges? Don't want to allow committing and pushing on these branches.